packaging/nnstreamer.spec

```diff
@@ -102,6 +105,22 @@ Requires:	nnstreamer = %{version}-%{release}
 %description tensorflow-lite
 NNStreamer's tensor_fliter subplugin of TensorFlow Lite.
 
+%package python-filter
```

Comment:
Let's name
```%package python2``` for python2 filters
and
```%package python3``` for python3 filters.

Reply:
Got it